include/cpr/api.h

```diff
@@ -51,6 +52,39 @@ void set_option(Session& session, Ts&&... ts) {
     set_option_internal<false, Ts...>(session, std::forward<Ts>(ts)...);
 }
 
+// Idea: https://stackoverflow.com/a/19060157
+template <typename Tuple, std::size_t... I>
+void apply_set_option_internal(Session& session, Tuple&& t, std::index_sequence<I...>) {
```

Comment:
We use camel case for methods.

Reply:
I have orientated myself here with the notation on the other, already existing set_option functions. Should I change the notation anyway?

Reply:
It looks like we are using lower case with '_' for template helper functions. For now this should then be fine, but in the future I will change this.